serverless-lambda-container-otel-layer-example/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "devDependencies": {
+    "serverless": "^3.30.1"
+  },
+  "name": "serverless-lambda-container-otel-layer-example",
+  "description": "Example Serverless + Lambda Container + Lambda Layer + OpenTelemetry integration",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "CodeSee Engineers",
+  "license": "MIT",
+  "dependencies": {
+    "aws-sdk": "^2.1389.0"
```

Comment:
Did you test that you were able to import the `aws-sdk` library?
It might be better to choose a different dependency that you can actually import, use, and verify is working. We know the `aws-sdk` library will be available no matter what. For example, you could use `bunyan`.

Reply:
yea I can simply print out its aws-sdk loaded version. Lambda default will be v3. I will also include bunyan as well.

Reply:
It might be simpler to just include and use bunyan and forget about aws-sdk :)